src/cogent3/app/io.py

```diff
@@ -451,12 +476,15 @@ class write_json(_checkpointable):
 
     _type = "output"
 
+    _input_types = "serialisable"
```

Comment:
and again, I don't know how so many of these were not changed to using the TYPEs